### PR TITLE
Add Teach Yourself Programming in Ten Years

### PR DESCRIPTION
**Pull request by diendang01**
A useful post for someone learning to program

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ But knowing the stuff will help you become better! :muscle:*
 - :book: [Cracking the Coding Interview: 189 Programming Questions and Solutions](https://www.goodreads.com/book/show/25707092-cracking-the-coding-interview)
 - :fire: [Everything you need to know to get the job](https://github.com/kdn251/interviews)
 - :book: [Tech Interview Handbook](https://github.com/yangshun/tech-interview-handbook)
+- :page_facing_up: [Teach Yourself Programming in Ten Years](http://norvig.com/21-days.html)
+
```
Comment (eamanu) [inline README.md]: I recommend take off this line... is just a suggestion

Comment (eamanu) [reply README.md]: The empty line

**Comment by eamanu** [comment]
Good suggestion: "Teach Yourself Programming in Ten Years"

**Comment by mr-mig** [comment]
Good stuff 👍

**Pull request merged**